pages/linux/qm-guest-passwd.md

```diff
@@ -0,0 +1,12 @@
+# qm guest passwd
+
+> Sets the password for the given user on QEMU/KVM Virtual Machine Manager.
```

Comment:
```suggestion
> Set passwords for the given user on QEMU/KVM Virtual Machine Manager.
```

Reply:
should it not be password instead of passwords since its user specific